website/content/docs/k8s/installation/install.mdx

```diff
@@ -199,6 +200,43 @@ NAME: consul
 ...
 ```
 
+#### Enable Consul Service Mesh on select namespaces
+
+By default, Consul Service Mesh is enabled on all namespaces within a Kubernetes cluster. You can restrict this to a subset of namespaces by specifying a `namespaceSelector` that matches a label attached to each namespace denoting whether to enable Consul service mesh.
```

Comment:
We exclude a couple by default: https://github.com/hashicorp/consul-k8s/blob/5027066696e2a95fea6342da96081c1581574149/charts/consul/values.yaml#L1797
```suggestion
By default, Consul Service Mesh is enabled on almost all namespaces within a Kubernetes cluster. You can restrict this to a subset of namespaces by specifying a `namespaceSelector` that matches a label attached to each namespace denoting whether to enable Consul service mesh.
```